src/index.js

```diff
@@ -0,0 +1,88 @@
+let description;
+let country;
+let temp;
+let cityname;
+let tempMetric;
+
+
+const convertTemp = (t) => {
+  const celcius = Math.floor(t - 273.15);
+  const farenheit = Math.floor((celcius * (9 / 5)) + 32);
+  return [celcius, farenheit];
```

Comment:
You should make different requests, please take a look at the `units` option of the Open Weather API: https://openweathermap.org/current#data

```
api.openweathermap.org/data/2.5/find?q=London&units=metric => Celsius 
api.openweathermap.org/data/2.5/find?q=London&units=imperial => Fahrenheit
```

Reply:
thanks for this detailed feedback. i will work on the changes. alex